### PR TITLE
Propose Release 0.6.3

### PR DESCRIPTION
**Pull request by jsturtevant**
I followed the docs at https://github.com/containers/oci-spec-rs/blob/main/release.md.  

In Particular I am interested in the `artifact_type` added in https://github.com/containers/oci-spec-rs/pull/145.  Thanks!

### CODE REVIEW

**Comment by utam0k** [comment]
@saschagrunert I agree with releasing the new version WDYT?

**Review by saschagrunert — APPROVED** [review]
Sounds good to me!

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "oci-spec"
-version = "0.6.2"
+version = "6.3.0"
```
Comment (saschagrunert) [inline Cargo.toml:3]: Should this be 
```suggestion
version = "0.6.3"
```

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This library provides a convenient way to interact with the specifications defin
 
 ```toml
 [dependencies]
-oci-spec = "0.6.2"
```
Comment (saschagrunert) [inline README.md:22]: Same here

**Review by saschagrunert — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ This library provides a convenient way to interact with the specifications defin
...
+oci-spec = "6.3.0"
```
Comment (saschagrunert) [inline README.md:22]:
```suggestion
oci-spec = "0.6.3"
```

**Review by saschagrunert — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (jsturtevant) [reply Cargo.toml:3]: oh woops!

**Comment by jsturtevant** [comment]
closing since dco is messed up with the extra commit. I re-ran the script and it created a new branch

**Pull request closed without merging**